dissect/target/tools/shell.py

```diff
@@ -719,6 +719,12 @@ def do_info(self, line: str) -> bool:
         print_target_info(self.target)
         return False
 
+    def do_reload(self, line: str) -> bool:
+        """reload the target"""
+        self.target = self.target.reload()
+        self.chdir(str(self.cwd))  # self.cwd has reference into the old target :/
```

Comment:
The typing on `self.cwd` looks to be wrong, I know we've talked about this many times but would you mind fixing it up a tiny bit? (Mostly just changing it from `str` to `Path`).

Reply:
Yes, sorry.